qir/qat/ValidationPass/ValidationPassConfiguration.cpp

```diff
@@ -67,9 +67,9 @@ namespace quantum
             profile.allow_internal_calls_     = false;
             profile.allowlist_external_calls_ = true;
             profile.allowlist_opcodes_        = true;
-            profile.opcodes_                  = Set{"ret",  "inttoptr", "br",   "phi", "add", "sub", "mul",  "fadd",
-                                   "fsub", "fmul",     "ashr", "and", "or",  "xor", "icmp", "fcmp"};
-            profile.external_calls_           = Set{
+            profile.opcodes_        = Set{"call", "ret",  "inttoptr", "br",  "phi", "add", "sub",  "mul", "fadd",
```

Comment:
Are these changes to the allowed opcodes for the other profiles also meant to be part of this PR?

Reply:
I did include this intentionally, yes. It was an oversight to leave "call" out of the list before.